front/pages/algorithms.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Algorithms = () => {
+  return <div>Algorithms</div>;
```

Comment:
You need to reconfigure the linter we don't wanna have those ";"
Take the liwi-medal-reader config 
Linter + prettify